.travis.yml

```diff
@@ -1,41 +1,55 @@
+# Use travis docker infrastructure
+sudo: false
 language: cpp
+
+env:
+    global:
+        - PREFIX=$HOME/prefix
+
 compiler:
-  - gcc
-#  - clang # not supported yet
+    - gcc
+    - clang
+
+matrix:
+    allow_failures:
+        # clang not supported yet
+        - compiler: clang
 
 # Install a recent gcc, gcov and cmake,
 # it will not be necessary once travis worker is based on ubuntu > 12.04.
 # Install SWIG for bindings generation
-before_install:
-  - sudo add-apt-repository --yes ppa:ubuntu-toolchain-r/test
-  - sudo add-apt-repository --yes ppa:kalakris/cmake # Non official cmake backport
-  - sudo apt-get update -qq
-  - sudo pip install cpp-coveralls
+addons:
+    apt:
+        sources:
+            - ubuntu-toolchain-r-test
+        packages:
+            - swig
+            - valgrind
+            - g++-4.8
 
 install:
-  - sudo apt-get install --yes swig cmake valgrind g++-4.8
-  - sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-4.8 100
-  - sudo update-alternatives --install /usr/bin/gcov gcov /usr/bin/gcov-4.8 100
-  - sudo wget --directory-prefix /usr/include/
+    - pip install --user cpp-coveralls; export PATH=$HOME/.local/bin:$PATH
+    - wget --directory-prefix $PREFIX
```

Comment:
> $ man cmake
> ***CMAKE_PREFIX_PATH***
>   Path used for searching by FIND_XXX(), with appropriate suffixes added.
> 
>   Specifies a path which will be used by the  FIND_XXX()  commands.  It  contains  the  "base"  directories,  the  FIND_XXX()  commands  append appropriate subdirectories to the base directories. So FIND_PROGRAM() adds /bin to each of the directories in the path, FIND_LIBRARY() appends /lib to each of the  directories,  and  FIND_PATH()  and  **FIND_FILE()  append  /include**  . By default it is empty, it is intended to be set by the project.

Catch is found with FIND_FILE. If I understand correctly the manual `cmake` looks for `catch.hpp` in `${CMAKE_PREFIX_PATH}/include`. Should you not download the header to `$PREFIX/include` ?